SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -9,8 +9,6 @@
 #include "SpatialConstants.h"
 #include "SpatialGDKSettings.h"
 
-#include <WorkerSDK/improbable/c_worker.h>
```

Comment:
Probably still want to include this no? 

We still use Worker_ConnectionParameters  which I assume exists inside c_worker.h. If so then we don't want to rely on other headers to include c_worker.h for us.

Reply:
Yup, that was a bogus delete.